### PR TITLE
[model]Enable qwen2

### PR DESCRIPTION
**Pull request by intellinjun**
## Type of Change

feature or bug fix or documentation or others
API changed or not

## Description

detail description 
Issues: xxx

## Expected Behavior & Potential Risk

the expected behavior that triggered by this PR 

## How has this PR been tested?

how to reproduce the test (including hardware information)

## Dependency Change?

any library dependency introduced or removed

### CODE REVIEW

File: neural_speed/models/qwen/qwen.h [hunk 1]
```diff
@@ -44,6 +44,18 @@ static const model_scratch qwen_mem_req(int n_layers, float scratch_size_ratio =
...
       };
+    case 28:
+      return {
+          static_cast<unsigned long long>(scratch_size_ratio * 4096) * MB,
+          static_cast<unsigned long long>(scratch_size_ratio * 2048) * MB,
+          static_cast<unsigned long long>(scratch_size_ratio * 4096) * MB,
+      };
+    case 80:
+      return {
+          static_cast<unsigned long long>(scratch_size_ratio * 10 * 4096) * MB,
```
Comment (a32543254) [inline neural_speed/models/qwen/qwen.h:55]: will times 10  compare to 28 layer is too large ?

**Review by Zhenzhong1 — APPROVED** [review]

File: neural_speed/models/qwen/qwen.h (hunk 1 above)
Comment (intellinjun) [reply neural_speed/models/qwen/qwen.h:55]: 7b->70B,so I used 10.

Comment (intellinjun) [reply neural_speed/models/qwen/qwen.h:55]: Qwen 70B have larger weight size than 7b,not only layers.

**Review by a32543254 — APPROVED** [review]
LGTM

**Pull request merged**